requirements.txt

```diff
@@ -0,0 +1,2 @@
+Django>=3.0,<4.0
+psycopg2-binary>=2.0
```

Comment:
Why do you have `psycopg2` if you're using sqlite? (though you should be using postgres)